config/configuration.json

```diff
@@ -413,5 +413,9 @@
       "difficulty": "NORMAL",
       "defaultPriority": "MEDIUM"
     }
+  },
+  "HighwayToFerryTagCheck": {
+    "enabled": true,
```

Comment:
This should not be included, as it can be obnoxious when trying to test other checks.

Reply:
It was added when testing. Have removed it.